community/server/src/main/java/org/neo4j/server/rest/transactional/Neo4jJsonCodec.java

```diff
@@ -71,6 +72,10 @@ else if ( value instanceof Map )
         {
             writeMap(out, (Map) value );
         }
+        else if( value instanceof ActsAsMap )
```

Comment:
This will never happen, when we get here it will be a `graphdb-api/spatial/point`